src/app/components/challenge/challengesubmit/challengesubmit.component.ts

```diff
@@ -418,6 +415,9 @@ export class ChallengesubmitComponent implements OnInit {
     FORM_DATA.append('method_description', self.globalService.formValueForLabel(self.components, 'method_description'));
     FORM_DATA.append('project_url', self.globalService.formValueForLabel(self.components, 'project_url'));
     FORM_DATA.append('publication_url', self.globalService.formValueForLabel(self.components, 'publication_url'));
+    if (self.validFileUrl === true && self.isSubmissionUsingUrl) {
```

Comment:
Don't you think you should also include this (https://github.com/Cloud-CV/EvalAI-ngx/pull/242/files#diff-7ebebddad833dbfe4a95492011383387R413) only when file upload is selected?

Reply:
i have put an if condition for file upload as requested